### ISSUE TITLE
Ember making a new view

### ISSUE THREAD

**Issue by doremaxime** [ga-wdi-boston/capstone-project#525]
The 1st view is the home page.
Goal => have a button in the nav which shows a new view (home page view is no longer displayed).
Status => I have generated a route which has created a file in templates, and one in routes. It has also added the ```  this.route('gears');``` in the router.
I believe the routes one is where the code for getting the data from the back end it.
I believe the template one is where the code for displaying the data is.

I have added a ```<h1>hello</h1>``` in the template and when i click on the nav button, it shows the h1 but under the home page data...

Do i need to generate something more? and why?

**Comment by doremaxime** [comment]
Still stuck.
I did ```ember g template gears/index``` and added some text for testing but now when I click on the nav button **My Gears** it show displays that there was an error:
![screen shot 2017-03-28 at 11 15 45 am](https://cloud.githubusercontent.com/assets/24608045/24412968/fcd85504-13a7-11e7-8837-2b2741ea9951.png)
Nothing is changing in the ember inspector so with my skill level I don't have hints to go off of.

**Comment by raq929** [comment]
Do you have an outlet in `routes/gears.js`?

**Comment by doremaxime** [comment]
Are we suppose to have outlets in js files?
You do mean ```{{outlet}}``` right?
Every example we've done we put models in the ```routes/some-file.js```...

**Comment by raq929** [comment]
Hah. Sorry. I have successfully linked the route and it's template in my mind! It should be in the `.hbs` file.  Without it, subroutes (like index) won't show up.

**Comment by doremaxime** [comment]
yes, there is an outlet in the hbs.

**Comment by doremaxime** [comment]
I have decided to not continue with ember.
No solution will be provided for this issue.

**Comment by jrhorn424** [comment]
This is the flash message displayed by the auth actions. If you're seeing that, it's usually some inconsistency with localStorage, so signing out and signing in usually fixes it.

Sorry to hear you've given up on Ember over this. Grepping through the codebase would have shown you where the error was coming from. Additionally, I didn't see your web console.

**Comment by doremaxime** [comment]
yeah saw that it was a flash message this morning but knowing that did not help me further. I thought I'd add it to the issue in case there was something about it I didn't know.

I spent from 8am till 4pm on this, i went over all the repos and the codebase because answers are there. However, I did not find it which is why I opened the issue. I know that Rachel was busy with the other cohort so I read over documentations, watched ember videos, and more. If all that plus the classes didn't help me for Ember I leaning towards ember not being for me. I believe it would be foolish to keep trying ember when there is 2 days left though. 

Respect for reading all issue uncle jeff :)
